docs/Readme.md

```diff
@@ -1,140 +1,115 @@
+branch.
 
-To generate a "requirements" file (usually requirements.txt), that you commit with your project, do:
+## 4 Prerequisites for building the Knora JSON API V1 / V2 Request and Response Format documentation
 
-```
-$ ./env/bin/pip freeze > requirements.txt
-```
+### 4.1 Requirements
 
-## 3 Install Graphviz ##
+The JSON request and response format is formally described using typescript interfaces. To create the docuemntation from these interfaces, we use `typedoc`.
 
-You will need [Graphviz](http://www.graphviz.org/). On Mac OS X:
+Install `typedoc` using `npm`:
 
 ```
-$ brew install graphviz
+npm install --global typedoc    
 ```
 
-On Linux, use your distribution's package manager.
-
-## 4 Generating Documentation ##
-
-### 4.1 Bulding the Documentation ###
+If you do not have `npm` (node package manager), install it first. You will find more information about `npm` here: <https://www.npmjs.com/>.
 
-Inside the docs folder type for the html documentation type
+## 5 Prerequisites for building Sphinx based documentation
 
-```
-$ make html
-```
+### 5.1 Install Python 3 ##
 
-or
+On Mac OS X:
 
 ```
-$ make latex
-$ make latexpdf
+$ brew install python3
 ```
 
-for generating a PDF. If there are errors during the latex compilation, skip them with 'R'. Also, run ```make latex``` a
-few times, so that the index is properly generated.
-
-The generated documentation can be found under  ```_build/latex```
-
-### 4.2 Problems with Locales ###
-
-Error message when trying to generate the documentation:
-
-```
-ValueError: unknown locale: UTF-8
-```
+On Linux, use your distribution's package manager, or see [the Python web site](https://www.python.org).
 
-Set the locales as environment variables (e.g. in ~/.bash_profile):
+### 5.2 Installing Sphinx ##
 
 ```
-export LC_ALL=de_CH.UTF-8
-export LANG=de_CH.UTF-8
+$ pip3 install sphinx
 ```
 
-## 5 Documentation of the Knora JSON API V1 Request and Response Format
-
-### 5.1 Requirements
-
-The JSON request and response format is formally described using typescript interfaces. To create the docuemntation from these interfaces, we use `typedoc`.
+### 5.3 Install Graphviz ##
 
-Install `typedoc` using `npm`:
+You will need [Graphviz](http://www.graphviz.org/). On Mac OS X:
 
 ```
-npm install --global typedoc    
+$ brew install graphviz
 ```
 
-If you do not have `npm` (node package manager), install it first. You will find more information about `npm` here: <https://www.npmjs.com/>.
+On Linux, use your distribution's package manager.
+
+## 6 Jekyll
 
-### 5.2 Building the Documentation
+### Installation
 
-Inside the docs folder type for the html documentation type
+Jekyll is used for building the *Knora Documentation Overview* website, which source can be found under `src/jekyll`.
+To install the necessary dependencies, run the following commands from inside the `src/jekyll` folder:
 
 ```
-$ make jsonformat
+$ gem install jekyll bundler
```

Comment:
Please add `jekyll-feed` here.

Reply:
did you run `bundle` from the `src/jekyll` directory? I think that this installs `jekyll-feed`

Reply:
No, I just did what it says in the description:

> inside the docs folder run sbt previewSite

I only realised later that there were more instructions in the README. :)